### PR TITLE
WFLY-13099 Standard configuration files for MicroProfile use cases

### PR DESCRIPTION
**Pull request by bstansberry**
https://issues.redhat.com/browse/WFLY-13099

### CODE REVIEW

File: testsuite/layers/pom.xml [hunk 1]
```diff
@@ -1622,6 +1622,94 @@
...
+                                    </layers>
+                                    <excluded-layers>
+                                        <layer>jpa</layer>
+                                    </excluded-layers>
+                                </config>
+                            </configurations>
+                        </configuration>
+                    </execution>
+                    <execution>
+>>>>>>> e9c00016f7... [WFLY-13099] Add layers tests for the combos used in standalone-microprofile.xml and standalone-microprofile-ha.xml
```
Comment (rhusar) [inline testsuite/layers/pom.xml]: Rebase oops? :-)

**Review by rhusar — CHANGES_REQUESTED** [review]

File: testsuite/layers/pom.xml (hunk 1 above)
Comment (bstansberry) [reply testsuite/layers/pom.xml]: Seems so!

**Comment by bstansberry** [comment]
retest this please

**Comment by bstansberry** [comment]
retest this please

**Pull request merged**